app/src/org/commcare/activities/CommCareSetupActivity.java

```diff
@@ -197,6 +216,43 @@ protected void onCreate(Bundle savedInstanceState) {
         }
     }
 
+    private BiometricPrompt.AuthenticationCallback preparePromptCallbacks() {
+        final Context context = this;
+        return new BiometricPrompt.AuthenticationCallback() {

```

Comment:
Pass these callbacks into the new BiometricsHelper method as an anonymous AuthenticationCallback